src/Stratis.Bitcoin.Features.SmartContracts.Tests/ExecutorSpecification.cs

```diff
@@ -12,6 +12,7 @@ public class ExecutorSpecification
         [Fact]
         public void Create_Contract_Success()
         {
+            // TODO: Fix
```

Comment:
Just check what's happening in the ContractExecutor is aligned with your thoughts and I'll update these